### DISCUSSION TITLE
Perfect score doesn't result in 100% of the pp

### DISCUSSION THREAD

**Discussion by MierenManz** [ppy/osu#26364] [Q&A]
![image](https://github.com/ppy/osu/assets/63878374/2ac564ea-9b11-4921-9afe-deb679e777b8)
This score is perfectly played. But the result page indicates that it got 63 out of the 74pp. Is this a bug that I should report or am I seeing something wrong?

**Comment by peppy** [comment]
Is this a play on the online leaderboard? If so then please ignore for now.

**Reply by MierenManz** [reply]
Yep online leaderboard
